### PR TITLE
Issues 56 and 57: Support executing PAL.ps1 from different directory and allow PS session log retention

### LINKED ISSUE #56: Regression: PAL.ps1 cannot be invoked unless current working directory is containing folder
Support for invoking PAL.ps1 from another directory was added when [PR#12](https://github.com/clinthuffman/PAL/pull/12) was completed, but was lost during the release of 2.8.1.

If you do not have your PowerShell session currently navigated to the directory that contains PAL.ps1, any attempts to invoke it will result in inherited threshold files to not be resolved and subsequently fail for various reasons.

### LINKED ISSUE #57: Regression: No flag for skipping clearing logs and errors
Support for invoking PAL.ps1 without clearing the entire PS session log and errors collection was added when [PR#12](https://github.com/clinthuffman/PAL/pull/12) was completed, but was lost during the release of 2.8.1.

If PAL is part of a larger automation script, it would be very valuable to have the option to not clear the entire log of execution up to the point of invocation, so that a user can see the full view of what their automation did.

### PR DESCRIPTION
**Pull request by VonOgre**
Reintroduced capabilities added originally in PR#12, but lost during the v2.8.1 release.

Added support for calculating relative paths to various threshold files based on the location of PAL.ps1 instead of the PowerShell session's current working directory. This allows for PAL to be executed from anywhere without issue, which is beneficial for larger automation.

Added support for NOT clearing the PowerShell session at the beginning of execution, so a bigger automation script doesn't lose everything prior to PAL starting.

Re-enabled the debug logging to the user's temp directory, but added a timestamp to the file name, so that each execution doesn't wipe out the previous log.

Resolves #56 
Resolves #57

### CODE REVIEW

File: PAL2/PALWizard/bin/Debug/PAL.ps1 [hunk 1]
```diff
@@ -916,9 +935,13 @@ Function InheritFromThresholdFiles
     #// Inherit from other threshold files.
     ForEach ($XmlInheritance in $XmlThresholdFile.SelectNodes('//INHERITANCE'))
     {
-        If ($(Test-FileExists $XmlInheritance.FilePath) -eq $True)
+        $inheritedXml = $($XmlInheritance.FilePath)
+        If(-not ([string]::IsNullOrWhiteSpace($inheritedXml))){
+            $inheritedXml = Resolve-Path "$PSScriptRoot\$inheritedXml"
```
Comment (VonOgre) [inline PAL2/PALWizard/bin/Debug/PAL.ps1]: Using $PSScriptRoot throughout makes the relative paths rooted to whatever the containing directory is for PAL.ps1

**Review by clinthuffman — APPROVED** [review]
Thank you for the assistance! Sorry for the delay, I never received a notification about this and just now found it.

**Comment by VonOgre** [comment]
No worries! I'd honestly completely forgotten that I had outstanding pull
requests so I was surprised to get emails today :P

**Comment by tardekar1997** [comment]
hello,
can u please give steps in short for adding new perf counters, bcoz in sql server 2014 threshold xml, physical disc counters (disc latency) are not present, i tried add them but failing, on internet also its not available

**Pull request merged**